src/Stratis.Bitcoin/BlockPulling/BlockPuller.cs

```diff
@@ -148,7 +148,7 @@ public class BlockPuller : IBlockPuller
         private const int AverageBlockSizeSamplesCount = 1000;
 
         /// <summary>The minimal count of blocks that we can ask for simultaneous download.</summary>
-        private const int MinimalCountOfBlocksBeingDownloaded = 10;
+        private const int MinimalCountOfBlocksBeingDownloaded = 10000;
```

Comment:
This looks like a logic change rather than fixing a warning.

Reply:
That's a local change that got committed inadvertently. Reverting it.

Reply:
That seems like a good addition